Neural Network/neural_network.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""Neural Network.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1HMxwkMbHBiP3DnVkR59bFIwQ7WLo8S51
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
matplotlib is not being used anywhere. Please omit this line and we are good to go!

Reply:
Done it